CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/uncollide/experimental/TreeFactory.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2010-2020 Australian Signals Directorate
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.constellation.plugins.arrangements.uncollide.experimental;
+
+import au.gov.asd.tac.constellation.graph.GraphReadMethods;
+
+
+/**
+ *
+ * @author Nova
+ */
+class TreeFactory {
```

Comment:
access modifiers

Reply:
Deliberate leaving this at default/package private access level.
If it were to be public it would need to reside somewhere different as a general graph utility class. Given the only use I can see for it is to find colliders I think it is very unlikely anything else will want to use this class.